package.json

```diff
@@ -15,7 +15,7 @@
     "test": "jest"
   },
   "engines": {
-    "node": ">=12"
+    "node": ">=14.13.1"
```

Comment:
![memo](https://img.shields.io/badge/review-memo-white.svg)
Removed support for Node.js 12.
https://stylelint.io/migration-guide/to-15#removed-support-for-nodejs-12